db/migrate/20161024222746_add_entry_id_to_entries.rb

```diff
@@ -0,0 +1,5 @@
+class AddEntryIdToEntries < ActiveRecord::Migration[5.0]
+  def change
+    add_column :entries, :entry_id, :string
```

Comment:
Looks like this needs to be rebased after #162 was merged — do you want to move this line up into the original migration to create `entries`?